lms/static/sass/elements/_controls.scss

```diff
@@ -416,3 +416,20 @@
       color: $link-color;
     }
   }
+
+.btn-link {
+    @extend %shame-link-text;
+
+    // reset of inherited buttons
+    border-radius: 0;
+    border-color: $transparent;
+    padding: 1px;
+    background: $transparent;
+    background-image: none;
+
```

Comment:
adding this to the reset list should resolve the size issue Christina noted.

@extend %t-action3; 
@extend %t-strong;